spec.html

```diff
@@ -52991,6 +53014,15 @@ <h1>Non-default behaviour in HostMakeJobCallback</h1>
       <h1>Non-default behaviour in HostEnsureCanAddPrivateElement</h1>
       <p>The HostEnsureCanAddPrivateElement abstract operation allows hosts which are web browsers to specify non-default behaviour.</p>
     </emu-annex>
+
+    <emu-annex id="sec-late-errors-for-function-call-assignment-targets">
+      <h1>Late Errors for Function Call Assignment Targets</h1>
```

Comment:
```suggestion
      <h1>Runtime Errors for Function Call Assignment Targets</h1>
```
Also gotta update the IDs.